package.json

```diff
@@ -33,12 +33,13 @@
     "gpsoauthnode": "0.0.5",
     "long": "^3.2.0",
     "node-pogo-protos": "^2.4.2",
-    "node-pogo-signature": "^4.0.0",
+    "node-pogo-signature": "pogosandbox/node-pogo-signature",
```

Comment:
We can't find a way to reach spencer. This is just temporary, @starkevin will create a new module on rpm.
I'll update to use starkevin's repo, I guess it's better than mine (fork of a fork...)